packages/mdc-chips/_mixins.scss

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "./variables";
+
+@mixin mdc-chip-border-radius($radius: $mdc-chip-border-radius-default) {
```

Comment:
It seems odd to me for the parameters to these public mixins to have default values (it's more arguable for multi-parameter mixins, but see comment below RE stroke). If someone is calling one of these mixins, they clearly have a value in mind to pass to it.

I'm also wondering if we want to provide something similar to mdc-button-filled-accessible for chips? (At least, if chips are typically expected to have black or white ink in many cases.)

Reply:
Makes sense, removed the default values and made them baseline arguments provided by `mdc-chip`.

The accessibility thing is a great idea! I added a `mdc-chip-accessible-color` mixin.